.github/workflows/release.yml

```
@@ -1,83 +1,123 @@
+          files: |
+            apps/electron-app/dist/*.dmg
+            apps/electron-app/dist/*.zip
+            apps/electron-app/dist/*.exe
+            apps/electron-app/dist/*.AppImage
+            apps/electron-app/dist/*.snap
+            apps/electron-app/dist/*.deb
+          draft: false
+          prerelease: false
+          generate_release_notes: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }} 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)